scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -383,6 +383,12 @@ declare namespace WebdriverIO {
         args: Array<string>
     }
 
+    type ViewMatcher = {
+        name: string,
+        args: Array<string | object>
+        class?: string
```

Comment:
Shouldn't be class a required property? Otherwise this would turn into a `DataMatcher` no?

Reply:
Yes. This is the reason of my breaking change in the 1st commit, both DataMatcher and ViewMatcher objects can look identical. The class field is an optional property on both DataMatcher and ViewMatcher, and similarly, the args field can be an array of either String or Object. The difference is on the Espresso side, DataMatcher calls 'onData' and ViewMatcher calls 'onView'.

Reply:
ah ok, so can we merge these types then and give them a more general name?